code/modules/projectiles/projectile/magic.dm

```diff
@@ -266,6 +268,9 @@
 	if(created)
 		return ..()
 	if(LAZYLEN(contents))
+		var/obj/structure/closet/decay/locker_temp_instance = locker_ref.resolve()
+		if(!locker_temp_instance)
```

Comment:
you shouldve just `if (!isnull(locker_ref.resolve()))`

Reply:
nah